CSharp/Samples/Microsoft.Bot.Connector.Teams.SampleBot.Shared/MessageProcessor.cs

```diff
@@ -136,7 +136,7 @@ private static async Task HandleTextMessages(Activity activity, ConnectorClient
                     },
                 };
 
-                await connectorClient.Conversations.SendToConversationAsync(newActivity, response.Id);
+                await connectorClient.Conversations.SendToConversationAsync(response.Id, newActivity);
```

Comment:
This is a breaking change.

Reply:
This is just for my education purpose. Is the reason why we need to flip the order of parameters because of some changes in bot framework v3 SDK?